hmns-app/server/index.ts

```diff
@@ -29,3 +31,14 @@ const startServer = async () => {
 };
 
 startServer().catch((error) => console.log(error));
+
+let randomButterflyId:string="653d56ec705f7a57e560d711"; //current BOTD
+//set new BOTD every midnight
+cron.schedule('0 0 * * *', function() {
+ const randomButterfly=ButterflyResolvers.Query.randomButterfly();
+ randomButterfly.then(function(result){
```

Comment:
What happens when there's an error? Can we catch it and handle it gracefully? Maybe attempt a maximum of three time and then log an error?